soweego/linker/feature_extraction.py

```diff
@@ -327,3 +328,123 @@ def intersection_percentage_size(pair):
             return count_intersect / count_total if count_total > 0 else np.nan
 
         return fillna(concatenated.apply(intersection_percentage_size), self.missing_value)
+
+
+class OccupationCompare(BaseCompareFeature):
+
+    name = "OccupationCompare"
+    description = "Compares occupations attribute of record pairs."
```

Comment:
Same comment as per the class name

Reply:
Do you mean that we should change the class name to `occupation_compare`? I think that recordlinkage actually uses Capitalized names for the features classes: [for example the Exact feature](https://github.com/J535D165/recordlinkage/blob/87a5f4af904e0834047cd07ff1c70146b1e6d693/recordlinkage/compare.py#L29-L55)

Reply:
> Do you mean that we should change the class name to `occupation_compare`?

Perhaps something more eloquent than `occupation_compare`. With respect to the snake case convention, yes.

> I think that recordlinkage actually uses Capitalized names for the features classes: [for example the Exact feature](https://github.com/J535D165/recordlinkage/blob/87a5f4af904e0834047cd07ff1c70146b1e6d693/recordlinkage/compare.py#L29-L55)

Nope, see https://github.com/J535D165/recordlinkage/blob/87a5f4af904e0834047cd07ff1c70146b1e6d693/recordlinkage/compare.py#L52

Not a big deal anyway

Reply:
I'm taking care of this super-minor comment